frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/useBuilderFields.tsx

```diff
@@ -32,45 +36,60 @@ const getFormFieldsWhileCreating = (
         _id: cuid(),
       })),
     }
-    return insertAt(formFields, fieldToCreate.insertionIndex, newField)
+    formFields.splice(fieldToCreate.insertionIndex, 0, newField)
   }
-  return insertAt(formFields, fieldToCreate.insertionIndex, {
-    ...fieldToCreate.field,
+  formFields.splice(fieldToCreate.insertionIndex, 0, {
```

Comment:
this may trigger double splice on table fields yea? should we return in the table block to prevent that from happening?

Reply:
fixed